core/server/data/migrations/versions/4.5/03-remove-unique-constraint-from-product-name.js

```diff
@@ -0,0 +1,16 @@
+const {logging} = require('../../../../../shared');
```

Comment:
I don't think this works because we don't re-export `logging` from within shared. But if you switch to the helper function like Naz suggested, it does the logging for you 🙂